src/ViewHoldingsRecord.js

```diff
@@ -229,19 +249,21 @@ class ViewHoldingsRecord extends React.Component {
     referenceTables.holdingsNoteTypes = holdingsNoteTypes.records;
 
     const query = location.search ? queryString.parse(location.search) : {};
-    const that = this;
-    const formatMsg = this.props.stripes.intl.formatMessage;
 
     const detailMenu = (
       <PaneMenu>
-        <IconButton
-          icon="edit"
-          id="clickable-edit-holdingsrecord"
-          style={{ visibility: !holdingsRecord ? 'hidden' : 'visible' }}
-          href={this.craftLayerUrl('editHoldingsRecord')}
-          onClick={this.onClickEditHoldingsRecord}
-          title={formatMsg({ id: 'ui-inventory.editHoldings' })}
-        />
+        <FormattedMessage id="ui-inventory.editHoldings">
+          {title => (
+            <IconButton
+              icon="edit"
+              id="clickable-edit-holdingsrecord"
+              style={{ visibility: !holdingsRecord ? 'hidden' : 'visible' }}
+              href={this.craftLayerUrl('editHoldingsRecord')}
+              onClick={this.onClickEditHoldingsRecord}
+              title={title}
```

Comment:
We shouldn't be using `title` attributes anymore: https://issues.folio.org/browse/STCOM-296
`ariaLabel` should work fine here instead.